news/2018-02-25-qat-gazette-february-2018.md

```diff
@@ -0,0 +1,81 @@
+
+We're still trying to determine an optimal strategy for evaluating aspiring Beatmap Nominators, balancing critical appraisal with accessibility and fairness for all. In the recent application round, the application pool was divided in two, with each half being evaluated by a team of three Quality Assurance Team members. Each team would attempt to come to a consensus on every applicant from their pool, based on experience, behaviour, modding quality, and so on.
+
+In an effort to remove bias, future application cycles will require each QAT to cast a **preliminary decision in private**, without input from other team members potentially swaying their decision. Only then will the examiners convene to share their opinions, with the QAT members from the second group then cross-examining the final decisions. Previous cycles utilised an "everyone checks everything" styled approach, but this sometimes led to huge workload strain and rushed decisions, so we feel this is an agreeable middle-ground.
+
+## Within the Quality Assurance Team
+
+Aside from continuing to develop Beatmap Nominator evaluations, there have been ongoing, small-scale developments in a number of other working fields such as the mode-specific Ranking Criteria rework, metadata checking, and mapping contests (the latter of which we'll address in a bit). What I'd like to focus on this month however, are two areas of recently increased significance that are long overdue some attention.
+
+### Acceptable content rule clarification
+
+A current hot topic in the osu! Ranking Criteria has been the unclear ruling on what we deem to be **"inappropriate or not-safe-for-work content"**. With the recent allowance/removal of some potentially borderline cases, it has become necessary to clarify and expand upon the current rule wording. This will aim to satisfy both mappers who wish to continue creating more mature-themed content, as well as players and moderators who wish to ensure osu! remains a safe and enjoyable environment for all.
+
+A first draft of the [rule change proposal](https://gist.github.com/Ephemeralis/b253dedf2fd87bbb6c154e483d449f2a) has been opened for a round of community discussion. Here, anyone can comment on the planned changes, as well as suggest further revisions. Included are guidelines on how to better decide if a song is appropriate for osu!, as well as clarification on what happens if a track is deemed to be unacceptable.
```

Comment:
probably want to change this to reference the fact that this got implemented and link to the thing in the ranking criteria or something

Reply:
Switched this out with the RC article and the fact that it's now finalised